lib/zclient.h

```diff
@@ -245,7 +246,8 @@ struct zapi_ipv6
 };
 
 extern int zapi_ipv6_route (u_char cmd, struct zclient *zclient, 
-                     struct prefix_ipv6 *p, struct zapi_ipv6 *api);
+                     struct prefix_ipv6 *p, struct prefix_ipv6 *src_p,
+                     struct zapi_ipv6 *api);
```

Comment:
The source-prefix will not be sent across the zserv/zclient wire if a standard dest-route is sent. To tell the srcdest-route case apart from the dst-route case on the receiving side, the flag is looked at.

On the sending side, one could infer the flag from src_p != NULL, however the current zclient api puts the responsibility of setting the right flags for the message on the caller, so it would feel a bit inconsistent to me if the flag is automatically determined for this one field, but not for any others.